source/type/type.cpp

```diff
@@ -262,21 +262,25 @@ ValidityPattern VehicleJourney::get_vp_for_section(
     return vp_for_section;
 }
 
-const std::pair<boost::optional<uint16_t>, boost::optional<uint16_t>> VehicleJourney::get_bounds_orders_for_section(
+std::set<StopPoint*> VehicleJourney::get_sections_stop_points(
```

Comment:
Don't you loose the order with a set? might raise problems for lollipop vp (RER C at Versailles), no?
How do you know that you should only impact first stop_time?